static/css/restyle/restyle.less

```diff
@@ -1774,6 +1773,11 @@ h3.author .transfer-ownership {
   .addon.hovercard & {
     border: 1px solid #ebebeb;
   }
+
+  .more-versions {
+    color: #0996f8;
+    margin: 0 .25em 0 0.20em;
```

Comment:
This is a nitpick but could you be consistent with the format here? I'm not sure if the file overall prefers or omits leading zeros, but please use/don't use them in both cases to be consistent :-)

Also, we seem to use `px` in most places, why use `em` here?